CondFormats/PhysicsToolsObjects/test/SiStripDeDx2DBuilder.cc

```diff
@@ -24,7 +24,7 @@ class SiStripDeDx2DBuilder : public edm::EDAnalyzer {
   virtual void analyze(const edm::Event&, const edm::EventSetup&);
 
 private:
-  edm::FileInPath fp_;
+  //edm::FileInPath fp_;
```

Comment:
NOTE: this was left intentionally commented out as the _code_ that uses `fp_` was already commented out.